### PR TITLE
ReconciliationUI: Display Source Product Gender & Style Changes

### PR DESCRIPTION
**Pull request by vahidbazzaz**
This PR updates the style and text of reconciliation UI, as well as correctly displaying source product gender.

### CODE REVIEW

**Comment by vahidbazzaz** [comment]
[2.0 Bug: #tiny ReconciliationUI does not display SourceProduct Gender](https://trello.com/c/xrOtJFWC/1320-20-bug-tiny-reconciliationui-does-not-display-sourceproduct-gender)

**Comment by vahidbazzaz** [comment]
[2.0 #tiny ReceiveOverlay Design Change for Usability](https://trello.com/c/AJ0dbWtH/1326-20-tiny-receiveoverlay-design-change-for-usability)

File: front/src/components/BoxReconciliationOverlay/components/MatchProductsForm.tsx [hunk 1]
```diff
@@ -155,14 +155,18 @@ export function MatchProductsForm({
...
         </Text>
         <Text
           fontSize={16}
-          fontWeight="semibold"
+          fontWeight={productId?.value === "" ? "semibold" : ""}
+          fontStyle="italic"
           style={{ color: productId?.value === "" ? "#FF0000" : "#000" }}
         >
           {shipmentDetail?.sourceProduct?.name}{" "}
+          {shipmentDetail?.sourceProduct?.gender
```
Comment (HaGuesto) [inline front/src/components/BoxReconciliationOverlay/components/MatchProductsForm.tsx:167]: Could you handle the none case here @vahidbazzaz ?

image

File: front/src/components/BoxReconciliationOverlay/components/MatchProductsForm.tsx [hunk 2]
```diff
@@ -155,14 +155,18 @@ export function MatchProductsForm({
     <form onSubmit={handleSubmit(onSubmitMatchProductsForm)}>
       <Flex direction="column" gap="2" alignItems="stretch">
```
Comment (HaGuesto) [inline front/src/components/BoxReconciliationOverlay/components/MatchProductsForm.tsx:156]: Nice to have: increase the gap to 4 and put a Box around the Product Gender part and one around the Size. Looks a bit nicer

**Review by HaGuesto — APPROVED** [review]
looks good. feel free to merge, when you addressed the comments

@vahidbazzaz besides my comment in the code, could you also check why the tests are failing?

File: front/src/components/BoxReconciliationOverlay/components/MatchProductsForm.tsx (hunk 1 above)
Comment (HaGuesto) [reply front/src/components/BoxReconciliationOverlay/components/MatchProductsForm.tsx:167]: @aerinsol

Comment (aerinsol) [reply front/src/components/BoxReconciliationOverlay/components/MatchProductsForm.tsx:167]: Yes drop the (none) in the no gender case

**Comment by vahidbazzaz** [comment]
> looks good. feel free to merge, when you addressed the comments
> 
> @vahidbazzaz besides my comment in the code, could you also check why the tests are failing?

@HaGuesto It seems only sometimes that tests fail in circleci, locally all tests pass

**Pull request merged**